### PR TITLE
A: apkdone.com (fixes https://github.com/AdguardTeam/AdguardFilters/i…

### PR DESCRIPTION
**Pull request by Yuki2718**
…ssues/87085)
https://github.com/AdguardTeam/AdguardFilters/issues/87085

### CODE REVIEW

File: easylist/easylist_specific_hide.txt [hunk 1]
```diff
@@ -842,6 +842,7 @@ experienceproject.com##.adn
 chat-avenue.com##.adnoyer_log
 flightglobal.com##.adp
 vice.com##.adph
+apkdone.com##.adplaceholder-top
```
Comment (spirillen) [inline easylist/easylist_specific_hide.txt:845]: Any particular reason you didn't suggest this as a generic hide?

**Comment by ryanbr** [comment]
`##.adplaceholder-top` would be better here @Yuki2718

File: easylist/easylist_specific_hide.txt (hunk 1 above)
Comment (Yuki2718) [reply easylist/easylist_specific_hide.txt:845]: It's shown on mobile too and uBO disables generic cosmetic by default on mobile. I'll switch to generic if requested, but then I'll add the specific filter to uAssets too.

**Comment by Yuki2718** [comment]
Done.

**Pull request merged**